src/frontend/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -37,31 +45,19 @@ export class VideoPlayer extends React.Component<
    * Noop out if the video or jwt is missing, render will redirect to an error page.
    */
   componentDidMount() {
-    const { video, jwt } = this.props;
+    const { video, jwt, getTimedTextTrackList } = this.props;
 
-    if (video && jwt) {
+    if (video) {
+      getTimedTextTrackList();
       // Instantiate Plyr and keep the instance in state
       this.setState({
         player: this.props.createPlayer('plyr', this.videoNodeRef!, jwt),
       });
 
-      // Use Shaka Player to stream using DASH ISO if the browser supports it
-      // Otherwise we'll fall back to HLS/MP4 as described in the <sources> of the <video> element
-      if (shaka.Player.isBrowserSupported()) {
-        // Install any necessary polyfills from the shaka package
-        shaka.polyfill.installAll();
-        // Instantiate the player, passing our <video> element
-        const shakaInstance = new shaka.Player(this.videoNodeRef!);
-        shakaInstance.configure({
-          abr: {
-            // Set a more sensible default bandwidth
-            // This lets us start with 480p instead of 144p with the default value
-            defaultBandwidthEstimate: 1600000,
-          },
-        });
-        // Pass our DASH manifest to Shaka
-        shakaInstance.load(video.urls.manifests.dash);
-      }
+      // defaultBandwidthEstimate: 1600000,
```

Comment:
I tried to apply this option on Dashjs but I didn't see where I can reuse it.

Reply:
The purpose is to make start with a medium quality video: avoid loading a trash resolution by default (which is ugly) or starting with HD (which is too slow for users with bad network).

I think `setInitialBitrateFor` might be the equivalent in dashjs.

Reply:
Done. How do you test the adaptive bitrate ? I try to enable throttling but the page doesn't load at all 😄

Reply:
This is supposed to only affect the first fragment, before there's anything to adapt to. If it loads a 480p fragment you're good.

Reply:
It loads a 480p fragment :-)

Reply:
Great!